Example/ViewController.swift

```diff
@@ -22,6 +23,39 @@ class ViewController: UITableViewController {
         }
     }
 
+    @IBAction func actionShowMiniApp() {
```

Comment:
Maybe `actionShowMiniAppById` instead so as to not get confused with showing a Mini App when clicking on the list item?

Reply:
you're right @corycaywood , I pushed a refactor